qa-mickey.planx-pla.net/manifest.json

```diff
@@ -45,6 +45,9 @@
     "with_s3": "yes",
     "with_cache": "no"
   },
+  "gateway": {
```

Comment:
Is this required for al qa envs? Also can we have a PR for qa-mickey separately first and then deploy to jenkins envs?

Reply:
sure